cargo-cyclonedx/src/generator.rs

```diff
@@ -372,65 +388,44 @@ pub enum GeneratorError {
 }
 
 fn top_level_dependencies(
-    member: &Package,
-    package_ids: &PackageSet<'_>,
-    resolve: &Resolve,
-) -> Result<BTreeSet<Package>, GeneratorError> {
+    member: &PackageId,
```

Comment:
This seems to be unused now. Rust is complaining.

Reply:
This is unused for now, but subsequent PRs make it used again. This is definitely the signature we'll need.

Reply:
I assumed as much but since you used the underscore for the `_resolve` but not here, I wasn't sure.
We can keep it as it is